libarchive/archive_parse_date.c

```diff
@@ -895,10 +912,7 @@ nexttoken(const char **in, time_t *value)
 		 * Because '-' and '+' have other special meanings, I
 		 * don't deal with signed numbers here.
 		 */
-		if (isdigit((unsigned char)(c = **in))) {
-			for (*value = 0; isdigit((unsigned char)(c = *(*in)++)); )
-				*value = 10 * *value + c - '0';
-			(*in)--;
+		if (consume_unsigned_number(in, value)) {
```

Comment:
Ah.  You moved that code from here and the original didn't have overflow checking either.  Shame on me.  ☹️

Reply:
This is still a valid issue nevertheless, and I have responded above.